src/routes/_header.svelte

```diff
@@ -25,6 +29,23 @@
 >
   <NamespaceSelect />
   <div class="flex flex-row justify-center items-center">
+    {#if $decryptPort}
+      {#if $lastDecryptStatus === null}
```

Comment:
Versus checking if something is null, should our logic be the opposite?

- `true`
- `false`
- `null`

I also kind of feel like having both `null` and `false` will eventually cause a bug like 9 months from now.

What if it _only_ could return `true` or `false` and we had another method for `$lastDecryptStatusExists` or something like that?

Reply:
```// An enum or string union may be more appropriate here so we dont' get a
// if thing and the falsiness evaluates to the null case, but start with this
export const lastDecryptStatus = writable<null | boolean>(null);```

I totally have this as a comment for this store XD. I agree probably an enum/string union would be best.

`'unrequested' | 'success' | 'error'`

Was kind of what I was thinking.